src/strings.js

```diff
@@ -1,26 +1,31 @@
-function sayHello (string) {
-  // your code here
+const sayHello = (string) => {
+let greeting = 'Hello, ';
+  return (greeting + string + "!")
```

Comment:
you've created the variable `greeting`, but you could have just used the string in your return statement. i.e.
```
return 'Hello, ' + string + '!';
```

Reply:
In this project, string concatenation is preferred. To use string concatenation to get the same result would look like:
```
return `Hello, ${string}!`;
```